GsaGH/Components/4_Analysis/AnalyseModel.cs

```diff
@@ -256,23 +257,23 @@ string message
               this.AddRuntimeError($" {message}");
             }
 
-            string report = model.Model.AnalysisTaskReport(task.Key);
-            string[] split = report.Split('\n');
-            report = split[0].Replace(": ", "\n") + "\n" + string.Join("\n", split.Skip(1));
-            report = report.Replace("Checking", "\nChecks:");
-            report = report.Replace("Solving", "\nSolver:");
-            report += "\n \n";
-            reports.Add(report);
+            string rep = report.Notes[task.Key];
```

Comment:
why are we shortening variable names?

Reply:
also this broke something..

you can´t access the notes with the task.Key! all current notes belong to the current task